src/Mvc/Mvc.Core/src/MvcCoreLoggerExtensions.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 #nullable disable
+#pragma warning disable CA1810 // Initialize all static fields inline.
```

Comment:
I'll try and do do this separately. The source generator does not like that we reuse ids.